src/Migrate/Migrate.Autorest/custom/Initialize-AzMigrateReplicationInfrastructure.ps1

```diff
@@ -241,21 +258,26 @@ public static int hashForArtifact(String artifact)
         foreach ($eachApp in $appMap.GetEnumerator()) {
             $SiteName = $eachApp.Value.Split("/")[8]
             $applianceName = $eachApp.Key
-            $HashCodeInput = $SiteName + $TargetRegion
 
             # User cannot change location if it's already set in mapping.
             $mappingName = "containermapping"
             $allFabrics = Get-AzMigrateReplicationFabric -ResourceGroupName $ResourceGroupName -ResourceName $VaultName
 
             foreach ($fabric in $allFabrics) {
                 if (($fabric.Property.CustomDetail.InstanceType -ceq "VMwareV2") -and ($fabric.Property.CustomDetail.VmwareSiteId.Split("/")[8] -ceq $SiteName)) {
+                    $HashCodeInput = $fabric.Id
```

Comment:
Changed storage account nomenclature to align with portal. We use appliance id to generate hash